src/checks/table_data_formatting_heirarchical.py

```diff
@@ -0,0 +1,30 @@
+from geopandas import GeoDataFrame
+
+from src.config import CheckReturnList
+
+
+def main(iso3: str, gdfs: list[GeoDataFrame]) -> CheckReturnList:
+    """Check completeness of an admin boundary by checking column hierarchy.
+
+    Args:
+        iso3: ISO3 code of the current location being checked.
+        gdfs: List of GeoDataFrames, with the item at index 0 corresponding to admin
+        level 0, index 1 to admin level 1, etc.
+
+    Returns:
+        List of check rows to be outputed as a CSV.
+    """
+    check_results = []
+    for admin_level, gdf in enumerate(gdfs):
+        row = {
+            "iso3": iso3,
+            "level": admin_level,
+            "total_heirarchical_columns": sum(
```

Comment:
I see this has the exact same check function as `levels_with_data` in `table_data_formatting_has_data.py`, so it should be safe to delete.

Reply:
Also fixed